nevergrad/optimization/optimizerlib.py

```diff
@@ -1060,7 +1061,7 @@ def intshare(n: int, m: int) -> Tuple[int, ...]:
             CMA(self.parametrization, num_workers=nw1),  # share parametrization and its rng
             TwoPointsDE(self.parametrization, num_workers=nw2),  # noqa: F405
             PSO(self.parametrization, num_workers=nw3),
-            SQP(self.parametrization, 1),  # noqa: F405
+            SQP(self.parametrization, num_workers=1),  # noqa: F405
```

Comment:
@teytaud FYI this is critical and can make a huge difference in ParaPorfolio and SQPCMA

Reply:
Waow!